janus_core/single_point.py

```diff
@@ -45,23 +83,32 @@ def __init__(
         self.set_calculator(**kwargs)
 
     def read_system(self, **kwargs) -> None:
-        """Read system and system name.
+        """
+        Read system and system name.
 
         If the file contains multiple structures, only the last configuration
         will be read by default.
+
+        Parameters
+        ----------
+        **kwargs
+            Keyword arguments passed to ase.io.read.
         """
         self.sys = read(self.system, **kwargs)
         self.sysname = pathlib.Path(self.system).stem
 
     def set_calculator(
```

Comment:
As in by defining a `@property` and `@[property].setter` for the calculator?

The calculator is an attribute of the `Atoms` object rather than `SinglePoint`, so how would that work?

Reply:
Sounds like a good reason to me.